### DISCUSSION TITLE
Dive Coaching Australia

### DISCUSSION THREAD

**Discussion by psa** [thuliteio/doks#947] [Showcase]
I've used a slightly customised (see pull requests) version of doks for https://www.divecoaching.com.au/

**Comment by h-enk** [comment]
NIce!

**Reply by h-enk** [reply]
I've added Dive Coaching Australia to the Doks Showcase: https://getdoks.org/showcase/dive-coaching-australia/

I hope that's ok with you?

**Comment by psa** [comment]
Of course! Thank you.
